src/docker-utils.coffee

```diff
@@ -218,7 +240,15 @@ do ->
 			.then ([ img ]) ->
 				knex('image').insert({ repoTag }) if !img?
 			.then ->
-				dockerProgress.pull(repoTag, onProgress)
+				Promise.join utils.getConfig('apiKey'), utils.getConfig('uuid'), (apiKey, uuid) ->
+					docker.getRegistryAndName(image)
+					.then ({ registry, imageName, tagName }) ->
+						dockerOptions =
+							authconfig:
+								username: 'd_' + uuid,
+								password: apiKey,
+								serveraddress: config.registryEndpoint
+						dockerProgress.pull(repoTag, onProgress, registry)
```

Comment:
I think this should be `dockerProgress.pull(repoTag, onProgress, { authconfig })`?